### ISSUE TITLE
(I am on)  for third or fourth segments (base_url/seg1/seg2/seg3/seg4) 

### ISSUE THREAD

**Issue by chhumsina** [Behat/Behat#686]
Hi friends,

I would like to ask about the (I am on).
My problem is that, It is always open browser to (base_url) when I put (I am on "base_url/seg1/seg2/seg3/seg4") but the script test show pass while the browser open base_url.

How to solve that or I have lacking point for that?

Thanks you so much :)

**Comment by everzet** [comment]
1. I don't understand what you are saying. Can you rephrase?
2. It seems your issue is with Behat/MinkExtension, not Behat/Behat itself. Can you reopen an issue with rephrased clear explanation there instead?

Thanks!
